gui/src/components/preboarding/welcome.vue

```diff
@@ -41,14 +41,22 @@
 <script lang="ts">
 import { Component, Vue, Prop } from "vue-property-decorator";
 import i18n from "./preboarding.json";
+import { unsupportedFeatures } from "../../common/unsupported-feature";
 
 @Component({
   name: "cortx-get-started",
   i18n: {
     messages: i18n
   }
 })
-export default class CortxWelcome extends Vue {}
+export default class CortxWelcome extends Vue {
+  public unsupportedFeatures = unsupportedFeatures;
+
+  private mounted() {
```

Comment:
Is this method required? And why is lifecycle hook method returning something?

Reply:
Removed this too as updated router file.